### DISCUSSION TITLE
How to start multiple/simultaenous timers

### DISCUSSION THREAD

**Discussion by iceryan354** [kimai/kimai#4078] [Questions & Answers]
I see in the settings that multiple time entries are possible at the same time, but it seems once i press the timer the only option is to press it to stop it timing. if I start job A at 12:30 and its going until 16:30 - but I am also doing job B from 14:30 to 15:30 what is the cleanest way to enter this in real time - not retroactively and not with fixed end times at entry - but just as the day goes on.... thanks! Ive looked for this, assuming it was a common question, so apologies if redundant. :)

**Comment by kevinpapst** [comment]
Just hit "+ Create" on the timesheet listing and do not enter a duration / end. 
That will create another running record.

**Reply by MichiFr** [reply]
Just tried it but this is not the best solution yet. 

In the demo account, I see more than one unfinished tasks on my dashboard and, as the @iceryan354 mentioned, I need to start two or more available tasks in parallel. 

Creating a new one each time for the same project or customer is not an option. This just adds additional complexity in managing my tasks.

Why not add an option in user settings that allows to choose starting tasks in parallel  or just a single one?

**Reply by kevinpapst** [reply]
If you are talking about Tasks (not timesheets), then this is exactly how it is working.
![image](https://github.com/user-attachments/assets/7a8e7ae7-657b-4db4-a5f2-d5cf5f8567fa)

Cool, right. You have to enable multiple timesheets under System > Settings
